src/Util/HTTPUtil.spec.ts

```diff
@@ -1,11 +1,38 @@
-import HTTPUtil from './HTTPUtil';
+import HTTPUtil, { PostOptions } from './HTTPUtil';
 
 describe('HTTPUtil', () => {
 
   describe('post', () => {
     it('is defined', () => {
       expect(HTTPUtil.post).toBeDefined();
     });
+    it('calls fetch with the expect params', async () => {
+      // @ts-ignore
+      fetch = jest.fn();
+      const headers = new Headers();
+      headers.set('Content-Type', 'application/x-www-form-urlencoded');
+      headers.set('superheader', 'header-abc');
+      // const expectedParams = {
+      //   headers,
+      //   method: 'POST',
+      //   credentials: 'same-origin',
+      //   body: new URLSearchParams({})
+      // };
```

Comment:
Minor: Please remove outcommented code

Reply:
This code belongs to the comment below. So i'd like to keep it.